k8s/object.bzl

```diff
@@ -488,8 +508,12 @@ def k8s_object(name, **kwargs):
         **implicit_args
     )
 
-    _k8s_object(name = name, **kwargs)
-    _k8s_object(name = name + ".resolve", **kwargs)
+    resolve_args = dict(**kwargs)
+    if "args" in resolve_args:
+        resolve_args.pop("args")
```

Comment:
I'm wondering if we should maybe just deprecate support for `args` and add a `kubectl_args` attribute so that it's clearer that this are only for the kubectl commands

Reply:
Makes sense but not totally sure it is worth it if others are already using args?

Can we document that `k8s_object` accepts `args` and/or `kubectl_args`?